frontend/src/scenes/actions/ActionEdit.tsx

```diff
@@ -136,127 +169,152 @@ export function ActionEdit({ action: loadedAction, id, onSave, temporaryToken }:
-                                    </a>
-                                </small>
+                                <Field name="slack_message_format">
+                                    {({ value, onValueChange }) => (
+                                        <>
+                                            <Input
+                                                addonBefore="Message format (optional)"
+                                                placeholder="Default: [action.name] triggered by [user.name]"
+                                                value={value}
+                                                onChange={(e) => {
+                                                    onValueChange(e.target.value)
+                                                }}
+                                                disabled={!slackEnabled || !action.post_to_slack}
+                                                data-attr="edit-slack-message-format"
+                                            />
+                                            <small>
+                                                <a
+                                                    href="https://posthog.com/docs/integrations/message-formatting/"
+                                                    target="_blank"
+                                                    rel="noopener noreferrer"
+                                                >
+                                                    See documentation on how to format webhook messages.
+                                                </a>
+                                            </small>
+                                        </>
+                                    )}
+                                </Field>
                             </>
                         )}
                     </div>
                 </div>
-                {errorActionId && (
-                    <p className="text-danger">
-                        Action with this name already exists.{' '}
-                        <a href={urls.action(errorActionId)}>Click here to edit.</a>
-                    </p>
-                )}
-                <div className="float-right">
-                    <span data-attr="delete-action-bottom">{deleteAction}</span>
-                    <Button
-                        disabled={!edited || !action.name}
+                <div style={{ display: 'flex', justifyContent: 'flex-end' }}>
+                    {!!id ? deleteButton() : cancelButton()}
+                    <LemonButton
                         data-attr="save-action-button"
                         type="primary"
-                        icon={<SaveOutlined />}
-                        onClick={saveAction}
-                        style={{ marginLeft: 16 }}
+                        htmlType="submit"
+                        disabled={!action.name}
```

Comment:
I think we should let users click save, but we tell them immediately why the save failed (eg "Action must have a name"). Otherwise, it can be confusing why the button is disbaled.